src/Http/Http.Extensions/gen/StaticRouteHandlerModel/Emitters/EndpointParameterEmitter.cs

```diff
@@ -44,12 +52,36 @@ internal static void EmitQueryOrHeaderParameterPreparation(this EndpointParamete
 
     internal static void EmitParsingBlock(this EndpointParameter endpointParameter, CodeWriter codeWriter)
     {
-        if (endpointParameter.IsParsable)
+        if (endpointParameter.IsArray && endpointParameter.IsParsable)
+        {
+            codeWriter.WriteLine($"{endpointParameter.Type.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat)} {endpointParameter.EmitHandlerArgument()} = new {endpointParameter.ElementType.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat)}[{endpointParameter.EmitTempArgument()}.Length];");
```

Comment:
Why not `EmitterConstants.DisplayFormat` here?

Reply:
Nothing other than ignorance (will be updated in an upcomin commit).